nengo/builder/probe.py

```diff
@@ -33,10 +33,8 @@ def synapse_probe(model, key, probe):
         raise ValueError("Attribute '%s' is not probable on %s."
                          % (key, probe.obj))
 
-    if isinstance(probe.slice, slice):
+    if probe.slice is not None:
         sig = sig[probe.slice]
-    else:
```

Comment:
That's because probing an ensemble makes a "connection_probe", as will probing neurons. Since this uses the Connection builder code, which supports advanced slicing, it works fine. It's only when you make a "synapse_probe" (the function in which the line in question is located), for example probing the voltages on neurons, that you run into problems with advanced slicing.